index.html

```diff
@@ -38,8 +41,190 @@ <h1>I'm Mucha <br> Glad to see <br> you!</h1>
+            </div>
+        </div>
+    </div>
+
+    <!-- card-4 -->
+    <div class="work-container4">
+        <div class="portfolio-img">
+            <img src="images/Snapshoot-Portfolio-4.png" alt="">
+        </div>
+
+        <div class="work-content">
+
+
+            <h1 class="tonic">Multi-Post<br>Stories</h1>
+
+            <ul class="prop">
+                <li class="can">CANOPY</li>
+                <li class="bac"><img src="images/Counter.png" alt=""> Back End Dev</li>
+                <li class="num"><img src="images/Counter.png" alt=""> 2015</li>
+            </ul>
+
+            <p class="tonic-def">A daily selection of privately<br> personalized reads; no accounts or<br> sign-ups
+                required.</p>
+
+            <ul class="stack">
+                <li class="html">html</li>
+                <li class="css">css</li>
+                <li class="js">javaScript</li>
+            </ul>
+
+            <div class="button">
+                <button type="button">See Project</button>
+            </div>
+        </div>
     </div>
 
+    <!-- about me  -->
+    <div class="introd">
+        <h1>About<br> Myself</h1>
+        <p>I'm a software developer! I can help you <br> build a product, feature or website Look <br> through some
+            of my work and <br> experience! If you like what you see and <br> have a project you need coded, don't
+            <br> hestiate to contact me
+        </p>
+
+        <a href="#" class="con">LET'S CONNECT</a>
+
+        <ul class="socials">
+            <li><a href="#"><img src="images/github.svg" alt="Github"></li>
+            <li><a href="#"><img src="images/Linkedin icon.svg" alt="linkedin"></a></li>
+            <li><a href="#"><img src="images/angellist.svg" alt="angellist"></a></li>
+            <li><a href="#"><img src="images/twitter.svg" alt="twitter"></a></li>
+            <li><a href="#"><img src="images/medium.svg" alt="medium"> </a></li>
+        </ul>
+
+        <div class="button">
+            <button type="button">Get my resume</button>
+        </div>
+
+        <div class="lang">
+            <div class="lang-head">
+                <h2>Languages</h2>
+                <img src="images/chev-down.png" alt="">
+            </div>
+
+            <ul class="lang-list">
+                <li class="script"><img src="images/Ellipse 1.png" alt="js"> JavaScript</li>
+                <li class="htm"><img src="images/Ellipse-html.png" alt="html"> HTML</li>
+                <li class="cs"><img src="images/Ellipse-css.png" alt="css"> CSS</li>
+            </ul>
+
+
+            <div class="frame">
+                <h2>Frameworks</h2>
+                <img src="images/Icon - Chevron - Right - Larger.png" alt="">
+            </div>
+
+            <img src="images/Separator Bottom.png" alt="">
+
+            <div class="skl">
+                <h2>Skills</h2>
+                <img src="images/Icon - Chevron - Right - Larger.png" alt="">
+            </div>
+
```

Comment:
- [ ] Good job creating all the required elements to match with the Figma template 👍, but for semantics, the technology categories ("Languages", "Frameworks" and "Skills") should be in a `ul`, and each of them in a `li` tag. And also the technology in each category(e.g., "JavasScript", "HTML" and "CSS") should be a list `ul` and each of them a `li` 💯